CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ since 0.4.1, and this project adheres to [Semantic Versioning](https://semver.or
   machine the Dangerzone uses under the hood
   ([#1172](https://github.com/freedomofpress/dangerzone/issues/1172))
 - Capture all the command outputs in the logs ([#1236](https://github.com/freedomofpress/dangerzone/issues/1172))
+- Allow Dangerzone to run in airgapped environments without extra configuration
```

Comment:
I don't think this needs a special entry in the changelog as it's part of the work we've done for ICU (dangerzone was already able to run in airgapped envs)